package.json

```diff
@@ -68,10 +70,11 @@
     "babel-plugin-module-resolver": "3.1.1",
     "concurrently": "3.6.0",
     "nodemon": "1.18.3",
-    "nyc": "12.0.2",
+    "nyc": "https://github.com/poetapp/nyc.git",
```

Comment:
Can you specify a commit hash? This way if changes are pushed the master branch of poetapp/nic they won't get picked up automatically by the node.

Reply:
yes, good idea!!!  :+1: